EIPS/eip-4974.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 4974
-title: Experience (EXP) Token Standard
-description: A standard interface for fungible, non-tradable tokens, also known as EXP.
+title: Comparable Non-Transferable Tokens
```

Comment:
For some reason the Github doesn't let me add the following suggestion to the Line 34.

Fixes the links to EIPs

```
--- [ERC-20](./eip-20) and [ERC-721](./eip-721)
+++ [ERC-20](./eip-20.md) and [ERC-721](./eip-721.md)
```

Reply:
Thanks, I'll update it.